src/tests/test_definitions.py

```diff
@@ -126,6 +126,16 @@ def nested_3(self):
 from __future__ import (, )
 """
 
+source_invalid_syntax = """
+while True:
+\ttry:
+    pass
+"""
+
+source_token_error = """
```

Comment:
Why do you need this to be a multi-line string? Why not just `source_token_error = "["`?